client/packages.go

```diff
@@ -51,36 +51,36 @@ const (
 	TypeOS        = "os"
 )
 
-// Packages returns the list of packages the system can handle
-func (client *Client) Packages() (map[string]*Package, error) {
-	const errPrefix = "cannot list packages"
+// Snaps returns the list of snaps the system can handle
```

Comment:
I did not express myself very well, sorry for that. I would prefer a sentence like the one Gustavo wrote or maybe something like:
```
Packages returns the list of all snaps installed on the system and available for install from the store for this system.
```
What I tried to say was that this is not the list of everything the system can handle because you can sideload package (and the meaning of "everything it can handle" was not obvious to me). But its not really important, its just a small detail (bike-shed land is very close :)